LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
No need for the license file. You can just refer to the MIT license in the README.